src/main/app/case/from-api-format.ts

```diff
@@ -15,6 +15,28 @@ const fields = {
     englishOrWelsh:
       data.LanguagePreferenceWelsh === YesOrNo.Yes ? LanguagePreference.Welsh : LanguagePreference.English,
   }),
+  D8DerivedPetitionerHomeAddress: data => {
```

Comment:
lets move this out to a function somewhere

Reply:
👍 Good idea, I've moved this out into a seperate file